addrbook.c

```diff
@@ -227,15 +224,11 @@ void mutt_alias_menu(char *buf, size_t buflen, struct Alias *aliases)
   menu->help = mutt_compile_help(helpstr, sizeof(helpstr), MENU_ALIAS, AliasHelp);
   mutt_menu_push_current(menu);
 
-new_aliases:
```

Comment:
We need this loop.
If you go to the alias selection menu, then add a new alias `:alias ...` it should show up in the menu.

Reply:
Dynamic adding new alias works only when *all* aliases are shown in the menu. But if the menu includes aliases selected by the completion than new alias will not be shown. It seems, this is the wrong place for this loop.

I'll restore functionality as it is in current `master`.